client/src/pages/SubmissionView.vue

```diff
@@ -0,0 +1,507 @@
+<template>
+  <div v-if="!submission" class="q-pa-lg">
+    {{ $t("loading") }}
+  </div>
+  <article v-else>
```

Comment:
I'm not sure these are semantically used.  The article tag around the submission content 👍, this one seems to be encapsulating more of the interface/chrome.

Reply:
Well, the interface/chrome contains content that constitute this part of the page as an independent, stand-alone piece of the site as defined by the HTML living standard. That standard also includes an example where the `header` is considered a child of its parent `article`: https://html.spec.whatwg.org/multipage/sections.html#the-article-element

This example demonstrates how a `header` and `footer` can be considered semantically part of their parent `article`: https://ilikekillnerds.com/2014/07/how-to-correctly-use-semantic-html5-article-main-and-section-tags/

This example demonstrates how an `aside` element can be considered semantically part of its parent `article`: https://html5doctor.com/aside-revisited/

Our implementation includes a `header` and an `aside` as part of the parent `article`.